File: 240320/Next Level/next-level.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String str = sc.next();
        int n = sc.nextInt();
        NextLevel n1 = new NextLevel();
        NextLevel n2 = new NextLevel(str, n);
        System.out.printf("user %s lv %d\n", n1.id, n1.level);
        System.out.printf("user %s lv %d", n2.id, n2.level);
    }
}

class NextLevel {
    String id;
    int level;

    public NextLevel() {
        this.id = "codetree";
        this.level = 10;
    }

    public NextLevel(String id, int level) {
        this.id = id;
        this.level = level;
    }
};

File: 240720/Next Level/next-level.js
const fs = require("fs");

//객체 선언
class NextLevel{
    constructor(id = "codetree", level = 10){
        this.id = id;
        this.level = level;
    }
}

const user1 = new NextLevel();

let [id, level] = fs.readFileSync(0).toString().trim().split(" ");
const user2 = new NextLevel(id, level);

console.log(`user ${user1.id} lv ${user1.level}`);
console.log(`user ${user2.id} lv ${user2.level}`);